src/types/settings.d.ts

```diff
@@ -31,4 +31,5 @@ export type Settings = {
   basePath: Path;
   translationEngines: Partial<TranslationEngines>;
   languages: Languages;
+  sort?: "A-Z" | "Z-A" | "none"; 
```

Comment:
sort can be an enum as Engines is:

https://github.com/heliomar-pena/i18n-populator/blob/8310b3231b65e24191446966ef3c98df9b921a48/src/types/settings.d.ts#L1-L5

then you can create another type based on that enum to represent any value of the enum:

https://github.com/heliomar-pena/i18n-populator/blob/8310b3231b65e24191446966ef3c98df9b921a48/src/types/settings.d.ts#L14-L17

Finally you can use the enum on JS instead of manually typing the name of each time you need it (`Sort.ALPHABETICALLY` for example)

https://github.com/heliomar-pena/i18n-populator/blob/8310b3231b65e24191446966ef3c98df9b921a48/src/services/translateService.ts#L33